src/utils/child-process.js

```diff
@@ -33,7 +33,7 @@ export function runCommand(cmd) {
     return new Promise((resolve, reject) => {
         const commands = cmd.split(SPACE);
         const [app, ...args] = commands;
-        const childProcess = spawn(app, args);
+        const childProcess = spawn(app, args, { stdio: 'ignore' });
```

Comment:
This change is acceptable since `runCommand` is not used to run docker container, just general utility commands.